Source/WebCore/style/StyleScopeRuleSets.cpp

```diff
@@ -344,6 +346,30 @@ const Vector<InvalidationRuleSet>* ScopeRuleSets::hasPseudoClassInvalidationRule
     return ensureInvalidationRuleSets(key, m_hasPseudoClassInvalidationRuleSets, m_features.hasPseudoClassRules);
 }
 
+const HashSet<AtomString>& ScopeRuleSets::customPropertyNamesInStyleContainerQueries() const
+{
+    if (!m_customPropertyNamesInStyleContainerQueries) {
+        HashSet<AtomString> propertyNames;
+
+        auto collectPropertyNames = [&](auto* ruleSet) {
+            if (!ruleSet)
+                return;
+            for (auto query : ruleSet->containerQueryRules()) {
```

Comment:
`const auto& query`

Reply:
(we don't need to increment the refcounter for a refcounted member variable IIRC)

Reply:
This hasn't been changed before the merge 😞

Reply:
const adds nothing here (rules are already const). Using references is less safe and since this is not performance critical there is no particular need to avoid churn.

At least my policy for cosmetic suggestions is that following them is at the discretion of the patch author.